src/components/View/commons/SideMenu/SideMenu.jsx

```diff
@@ -171,7 +171,7 @@ const SideMenu = ({ data, content_uid }) => {
                   }}
                   aria-controls="side-menu-body"
                 >
-                  <h3>{intl.formatMessage(messages.index)}</h3>
+                  <h2 className="h3">{intl.formatMessage(messages.index)}</h2>
```

Comment:
https://iosanita-staging.c3p0.redturtle.it/novita/comunicati/cento-alla-pandurera-lo-spettacolo-per-prevenire-la-dipendenza-da-gioco-dazzardo#:~:text=Condividi-,Indice%20della%20pagina,-Cos%27%C3%A8